podjs/src/browserPod.ts

```diff
@@ -312,7 +312,7 @@ class BrowserNetwork implements Network {
                     resolve(`Unexpected response status: ${status}`);
                     return;
                 }
-                resolve();
+                resolve("");
```

Comment:
This was erroring due to a change introduced in TypeScript 4.1. It was not possible to change the promise type, so I'm just returning an empty string.

Reply:
Hm, so TypeScript doesn't allow implicity `undefined` anymore? It still _should_ be `undefined` I would argue. In JavaScript it's kinda bad form to actually use the symbol `undefined` directly, I think in TypeScript it's fine though.

Reply:
Well, if I got this correctly, you need to return the type of promise you've declared, which is why I've used the same thing here. Returning undefined might imply to change the declaration of Promise, which is why I tried to solve it using the minimal change that's possible. I'll take a look anyway to see if that can be done.
Additionally, the problem is that podjs does not have a single test, so it's a bit build and then look around. Possibly adding some tests would be a good idea, since this is actually the main thing we use for local testing.

Reply:
Well, the signature of the function is quite clear: A string or `undefined`,  where a string means it was successful, and `undefined` means it wasn't. It happens to accidentally also work with an empty string cause that's fals-y, but it's still an incorrect implementation of `httpPost` :D